### DISCUSSION TITLE
Games Without Achievements Improvements

### DISCUSSION THREAD

**Discussion by Tsearo** [RetroAchievements/RAWeb#1506] [Ideas]
I think it would be beneficial to have the Games without achievements view to include more relevant information when this dropdown is selected as well as remove or change some of the current info that doesn't provide any useful details

Keep
- Leaderboards is still relevant as games can have these without achievements
- Open Tickets is also still relevant since sets can get taken down for tickets and it helps to track those cases

Remove
- Points makes no sense to include if there are no achievements in core

Change or Remove
- Achievements is always 0 but there could be some value in changing this to Unofficial Achievements in order to see which sets have something in unofficial
- Last Updated is currently never displayed though could  be useful to repurpose and pull info from Unofficial achievements

Add
- Claimed By 
- Number of Requests
Both of these are currently in the Most Requested Sets page but I think they would be useful information to see when looking through full game lists per system

**Comment by Jamiras** [comment]
Most of this can be managed by toggling the column visibilities. 

One could argue that the visibilities should automatically change based on the "Has Achievements" filter, but I think that would be hard to do well as long as the user can change the visibilities.

"Number of Requests" is currently not available as a column.
